host/examples/rx_ascii_art_dft.cpp

```diff
@@ -169,20 +170,115 @@ int UHD_SAFE_MAIN(int argc, char *argv[]){
             ascii_art_dft::log_pwr_dft(&buff.front(), num_rx_samps)
         );
         std::string frame = ascii_art_dft::dft_to_plot(
-            lpdft, COLS, LINES,
+            lpdft, COLS, LINES-5,
             usrp->get_rx_rate(),
             usrp->get_rx_freq(),
             dyn_rng, ref_lvl
         );
+        
+        std::string border = std::string((COLS), '-');
 
         //curses screen handling: clear and print frame
         clear();
+        printw("%s", border.c_str());
```

Comment:
It would be nice to add an option to print the controls. Something like:
```suggestion
        if (show_controls){
            printw("%s", border.c_str());
            ...
        }
```
This block put around the `printw`'s here. And also add the following to the argument parsing:
```
bool show_controls;
("show-controls", po::value<bool>(&show_controls)->default_value(true), "Show the keyboard controls")
```
and the window setup (L#173) as
```
lpdft, COLS, (show_controls ? LINES-5 : LINES),
```